### DISCUSSION TITLE
Limit CPU Usage

### DISCUSSION THREAD

**Discussion by mplorentz** [borgbase/vorta#809] [Ideas]
**Is your feature request related to a problem? Please describe.**
Running Vorta on my MacBook Pro causes the fans to spin up and my computer to get hot.

**Describe the solution you'd like**
I'd like a way to limit the cpu usage of Vorta/borg, resulting in a quieter, cooler, less noticeable backup.

**Describe alternatives you've considered**
There are some command-line tools (`nice`, `cputhrottle`) but I can't find a way to automatically run them every time Vorta starts a backup.

**Comment by rbirkelbach** [comment]
Yes, I have the same problem, too on a mbp 15,1 with a 2.3ghz 8core i9 processor and 16gb ram.
